docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -1,13 +1,13 @@
 [[configure-endpoint-integration-policy]]
-= Configure an integration policy for {endpoint-cloud-sec}
+= Configure an integration policy for {elastic-defend}
 
-After the {agent} is installed with the {endpoint-cloud-sec} integration, several protections features — including
+After the {agent} is installed with the {elastic-defend} integration, several protections features — including
 preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled
 on protected hosts (some features require a Platinum or Enterprise license). If needed, you can update the
 integration policy to configure protection settings, event collection, antivirus settings, trusted applications,
 event filters, host isolation exceptions, and blocked applications to meet your organization's security needs.
 
-You can also create multiple integration policies to maintain unique configuration profiles. To create an additional {endpoint-cloud-sec} integration policy, go to **Management** -> **Integrations**, then follow the steps for <<add-security-integration, adding the {endpoint-cloud-sec} integration>>.
+You can also create multiple integration policies to maintain unique configuration profiles. To create an additional {elastic-defend} integration policy, go to **Management** -> **Integrations**, then follow the steps for <<add-security-integration, adding the {elastic-defend} integration>>.
```

Comment:
Not sure if we should refer to our policies as `{elastic-defend} integration policy`, `integration policy`, or if it matters much. But in the first sentence it is sans {`elastic-defend}`; in the second it is not.

Reply:
Both sentences would be good with `{elastic-defend} integration policies/policy`, for clarity and consistency; will make that change!

Reply:
```suggestion
You can also create multiple {elastic-defend} integration policies to maintain unique configuration profiles. To create an additional {elastic-defend} integration policy, go to **Management** -> **Integrations**, then follow the steps for <<add-security-integration, adding the {elastic-defend} integration>>.
```